utils/httpUtils.ts

```diff
@@ -27,6 +27,23 @@ export const get = async (url: string, args: any, context: any) => {
     }
 };
 
+export const simpleGet = async (url: string, context: any) => {
+    try {
+        const baseURL = process.env.API_URL;
+        const response = await fetch(baseURL + url, {
+            method: 'GET',
+            headers: {
+              'Cookie': context.request.headers.get("cookie"),
+              'Content-Type': 'application/json',
+            },
+          });
+        return await response.json();
+    } catch (e) {
+        console.log(e);
```

Comment:
(nit) could we use `console.error`?  I don't know if we have linting rules setup, but typically we'd avoid having `console.log` in prod